### ISSUE TITLE
Navigation links showing active link via presentation but has no corresponding markup for screen readers

### ISSUE THREAD

**Issue by neilmispelaar** [DTS-STN/Alpha-Site#229]
### Description

The Navigation links (in this example Sign Up) is underlined (conveying meaningful information via presentation) showing it is the current page the user is on, but in code there is no hidden text telling the user the same information (that this link is the link that the user is currently visiting).

### How to Fix
Simply need to add some hidden text to the active link such as:

```
<a><span class="sr-only">You are currently on this page: </span>Sign Up</a> 
```

### WCAG Compliance

This constitutes a fail of WCAG 1.3.1 
https://www.w3.org/TR/UNDERSTANDING-WCAG20/content-structure-separation-programmatic.html

### Screenshot

<img width="1590" alt="Screen Shot 2021-07-21 at 1 39 10 PM" src="https://user-images.githubusercontent.com/9325038/126534559-24897e15-0fa7-419c-aad8-2a85ed6441fc.png">

**Comment by pcraig3** [comment]
I think it's easier then adding that hidden text. I had a check what [Bootstrap does with its navigation menu](https://getbootstrap.com/docs/5.0/examples/navbars/#), and they add `aria-current="page"` to the currently-active link. 

So, we could make sure that's on the links without adding a sentence of hidden text.

**Comment by neilmispelaar** [comment]
You definitely could do it with aria labels, the risk is that any assistive technologies that don't support ARIA roles won't announce anything. 

But from a compliance perspective, either technique nets a pass, it's just real world coverage would be increased using the hidden text technique.

Some notes on the aria-current values: 

- If on the current page: `aria-current="true"`
- If in the current set of pages (but not on the current page): `aria-current="page"`

#### Additional notes: 
https://www.w3.org/WAI/tutorials/menus/structure/#indicate-the-current-item
https://developer.mozilla.org/en-US/docs/Web/API/Element/ariaCurrent
https://www.accessibility-developer-guide.com/examples/sensible-aria-usage/current/#possible-values

**Comment by pcraig3** [comment]
Thanks for the links, @neilmispelaar! It looks like it would be best to preceding text, as you mentioned, but I don't want to create a dodgy French translation, so I'm shipping the "aria-current" change for now.
